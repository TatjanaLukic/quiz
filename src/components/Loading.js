import React from "react";
import '../index.css';

const Loading = () => {
  return <h1 className = "Loading">loading...</h1>;
};

export default Loading;
