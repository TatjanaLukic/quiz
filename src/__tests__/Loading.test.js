import React from 'react';

import { configure, shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Adapter from 'enzyme-adapter-react-16';

import Loading from '../components/Loading';

configure({adapter: new Adapter()});

describe('<Home/>',() => {

it("spanshot matches", () => {
    const tree = shallow(<Loading />);
    expect(toJson(tree)).toMatchSnapshot();
  });

});