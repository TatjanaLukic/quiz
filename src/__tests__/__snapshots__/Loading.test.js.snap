// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Home/> spanshot matches 1`] = `
<h1
  className="Loading"
>
  loading...
</h1>
`;
